node/chainSimulator/components/bootstrapComponents.go

```diff
@@ -50,17 +50,16 @@ func CreateBootstrapComponents(args ArgsBootstrapComponentsHolder) (*bootstrapCo
 	args.PrefsConfig.Preferences.DestinationShardAsObserver = args.ShardIDStr
 
 	bootstrapComponentsFactoryArgs := bootstrapComp.BootstrapComponentsFactoryArgs{
-		Config:                           args.Config,
-		PrefConfig:                       args.PrefsConfig,
-		ImportDbConfig:                   args.ImportDBConfig,
-		FlagsConfig:                      args.FlagsConfig,
-		WorkingDir:                       args.WorkingDir,
-		CoreComponents:                   args.CoreComponents,
-		CryptoComponents:                 args.CryptoComponents,
-		NetworkComponents:                args.NetworkComponents,
-		StatusCoreComponents:             args.StatusCoreComponents,
-		NodesCoordinatorWithRaterFactory: nodesCoordinator.NewIndexHashedNodesCoordinatorWithRaterFactory(),
-		RunTypeComponents:                components.GetRunTypeComponents(),
+		Config:               args.Config,
+		PrefConfig:           args.PrefsConfig,
+		ImportDbConfig:       args.ImportDBConfig,
+		FlagsConfig:          args.FlagsConfig,
+		WorkingDir:           args.WorkingDir,
+		CoreComponents:       args.CoreComponents,
+		CryptoComponents:     args.CryptoComponents,
+		NetworkComponents:    args.NetworkComponents,
+		StatusCoreComponents: args.StatusCoreComponents,
+		RunTypeComponents:    components.GetRunTypeComponents(),
```

Comment:
could this also be taken from the args? it might be more "configurable" for the future.

Reply:
This is chain simulator for node and it was made like this after the rc merge.
In the sovereign chain simulator it will be taken from args, but the change is not here, it in another PR

Reply:
ok